src/test/java/de/butbetter/keksys/CalculationControllerTests.java

```diff
@@ -17,4 +22,14 @@ void contextLoads() {
         assertThat(calcuationController).isNotNull();
     }
 
+    @Test
+    void sendParameter14and100() {
+        assertEquals(this.calcuationController.calculation(100, 14).getStatusCode(), HttpStatus.OK);
+    }
+
+    @Test
+    void sendParameterMinus1and100() {
+        assertEquals(this.calcuationController.calculation(-1, 100).getStatusCode(), HttpStatus.OK);
+    }
+
```

Comment:
Add test for the case that, the calculator throws exception in case of infinite result

Reply:
Will be done after being put on stage and thrown together

Reply:
Same as before